### PR TITLE
Updated request on city page to select only necessary fields

### PR DESCRIPTION
**Pull request by andrii52**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.BLL/Handlers/CityHandlers/GetCityAdminsHandler.cs [hunk 1]
```diff
@@ -1,43 +1,75 @@
...
-        public GetCityAdminsHandler(IMediator mediator)
+        public GetCityAdminsHandler(IRepositoryWrapper repoWrapper, IMapper mapper)
         {
-            _mediator = mediator;
+            _repoWrapper = repoWrapper;
+            _mapper = mapper;
         }
 
-        public async Task<CityProfileDTO> Handle(GetCityAdminsQuery request, CancellationToken cancellationToken)
+        public async Task<CityAdministrationViewModelDTO> Handle(GetCityAdminsQuery request, CancellationToken cancellationToken)
```
Comment (voroniak) [inline EPlast/EPlast.BLL/Handlers/CityHandlers/GetCityAdminsHandler.cs:28]:
```suggestion
        public async Task<CityAdministrationViewModelDTO> HandleAsync(GetCityAdminsQuery request, CancellationToken cancellationToken)
```

File: EPlast/EPlast.BLL/Handlers/CityHandlers/GetCityByIdWthFullInfoHandler.cs [hunk 2]
```diff
@@ -1,4 +1,7 @@
-﻿using System.Threading;
+﻿using System;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
 using System.Threading.Tasks;
 using AutoMapper;
 using EPlast.BLL.DTO.City;
```
[the remark this answers is not in the document]
Comment (andrii52) [reply EPlast/EPlast.BLL/Handlers/CityHandlers/GetCityByIdWthFullInfoHandler.cs]: Done

File: EPlast/EPlast.BLL/Handlers/CityHandlers/GetCityAdminsHandler.cs (hunk 1 above)
Comment (andrii52) [reply EPlast/EPlast.BLL/Handlers/CityHandlers/GetCityAdminsHandler.cs:27]: I can't rename this, the name is comming from interface that is part of the MediatR library.

File: EPlast/EPlast.BLL/Handlers/CityHandlers/GetCityAdminsHandler.cs [hunk 3]
```diff
@@ -1,43 +1,75 @@
...
+using System.Linq.Expressions;
+using System.Threading;
 using System.Threading.Tasks;
+using AutoMapper;
 using EPlast.BLL.DTO.City;
 using EPlast.BLL.Handlers.CityHandlers.Helpers;
 using EPlast.BLL.Queries.City;
+using EPlast.DataAccess.Entities;
+using EPlast.DataAccess.Repositories;
 using MediatR;
```
[the remark this answers is not in the document]
Comment (andrii52) [reply EPlast/EPlast.BLL/Handlers/CityHandlers/GetCityAdminsHandler.cs]: Done, thanks

**Review by bkondrat — APPROVED** [review]

**Review by Omeliukh-Nazar — APPROVED** [review]

**Review by Goven112 — APPROVED** [review]

**Pull request merged**